d3a_api_client/redis_client_base.py

```diff
@@ -251,9 +252,12 @@ def _on_market_cycle(self, msg):
         self.executor.submit(execute_function_util, function=function,
                              function_name="on_market_cycle")
 
-    def _on_event_or_response(self, msg):
-        message = json.loads(msg["data"])
-        logging.info(f"A new message was received. Message information: {message}")
+    def _on_event_or_response(self, log_msg):
+        message = json.loads(log_msg["data"])
+        log_msg = copy(message)
+        if 'grid_tree' in log_msg:
+            log_msg['grid_tree'] = '{...}'
```

Comment:
`log_msg['grid_tree'] = '{...}'` why would you log this  value `{...}`?

Reply:
Popping it sounds nice , I vote for it

Reply:
@hannesdiedrich I think this suggestion wasn't addressed (in case you wanted to use `pop()`).